### ISSUE TITLE
Precondition failed

### ISSUE THREAD

**Issue by ihitsov** [onokje/node-red-contrib-tesla#76]
Dear,

Last week my node-red tesla flow stopped working, without any intervention. I tried reverting a backup, but its no help. The node says "Precondition Failed". When I try to open the credentials of the node to replace the refresh token, node red crashes:
30 Jan 11:34:31 - [error] UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "Precondition Failed".

Do you have any idea how I can try to resolve this? I tried uninstalling, changing the refresh token etc, but I always end up in the same spot. Thanks in advance for your help!

Best regards,
Ivaylo

**Comment by fascenda** [comment]
I'm new to Node-RED and encountered this same problem, in issue #67 this is being discussed.  Tesla has removed their old API and opened a new one.  I know very little about it (I don't even have my Tesla yet), but I think the former method of using the tokens will no longer work.  I'm not sure if this can be solved, I'd say take a look at that discussion for any updates.

**Comment by ihitsov** [comment]
Yes I saw the other threads after posting mine. Thanks for pointing it out. Fingers crossed for a solution as I really liked how it worked before.

Best regards,
Ivaylo

**Comment by Bebeone111** [comment]
Any news on this? For me, the api is still not working.

**Comment by Michael-K-at-GitHub** [comment]
I propose closing this issue as a duplicate of #67
